### PR TITLE
ISPN-6427  Add test for multiple indexed caches using InfinispanIndex…

### PR DESCRIPTION
**Pull request by gustavonalle**
…Manager

https://issues.jboss.org/browse/ISPN-6427

### CODE REVIEW

File: client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/TwoCachesSharedIndexTest.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      for (RemoteCacheManager rcm : clients) {
+         MarshallerRegistration.registerMarshallers(ProtoStreamMarshaller.getSerializationContext(rcm));
+      }
+   }
+
+   private String read(String classPathResource) throws IOException {
+      return Util.read(getClass().getResourceAsStream(classPathResource));
+   }
+
+   @Test
```
Comment (anistor) [inline client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/TwoCachesSharedIndexTest.java]: I think the annotation in not required

Comment (slaskawi) [inline client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/TwoCachesSharedIndexTest.java]: The other test also has it, so I think it is alright...

Pulling

**Comment by slaskawi** [comment]
@gustavonalle Shall I push it to both `master` and `8.2.x`?

**Comment by slaskawi** [comment]
Integrated into master. Please create another PR for 8.2.x if needed.

**Comment by anistor** [comment]
@slaskawi Did you push it? I do not see it upstream yet.

**Comment by anistor** [comment]
I have cherry-picked this in 8.2.x too.

**Pull request closed without merging**